vagrant/ansible/roles/samba.setup/tasks/cephfs/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Install samba and samba-vfs-cephfs packages
+  yum:
+    name: samba-vfs-cephfs
+    state: present
```

Comment:
As per the Samba share configuration we don't yet make use of CephFS VFS module which makes the above installation optional.

Reply:
Yes, it was a left over from the initial attempt I made. I'll remove it and add support for CephFS VFS in another PR.